packages/core/src/components/input/input.stories.ts

```diff
@@ -55,23 +55,31 @@ export const Disabled = htmlMatrix(Input, { disabled });
 Disabled.argTypes = omit<InputProps>('disabled');
 
 interface InputWithLabelAndHelperTextProps extends InputProps {
+  id: string;
   label: string;
   helperText: string;
 }
 
 export const WithLabelAndHelperText = ({
+  id,
   label,
   helperText,
   ...props
 }: InputWithLabelAndHelperTextProps) =>
   Field({
     children: [
       FieldLabel({
-        children: [label, HelperText({ children: helperText }), Input(props)],
+        children: [
+          label,
+          HelperText({ children: helperText }),
+          Input({ ...props, id }),
```

Comment:
I don't think this line needs to change if we don't destructure the `id` 🤔

Reply:
Ah nvm you're using it for `for` later on, better than `props.id` I suppose.

Reply:
Yes, I do not like using `props.<something>` that much...

Reply:
Me neither